lib/IRGen/GenMeta.cpp

```diff
@@ -2906,9 +2906,6 @@ namespace {
       // Lay out the template data.
       super::layout();
       
-      // Save a slot for the field type vector address to be instantiated into.
-      asImpl().addFieldTypeVectorReferenceSlot();
```

Comment:
@rjmccall @jckarter What is this actually used for?

Reply:
Maybe also Mirror?  I think the NTD has an offset to it.  It would be great to remove that, of course.

Reply:
It's used by the field type vector instantiation code. I'm leaving it in place, it was just a speculative attempt at removing it which blew up tests.